highlighters/main/test-data/parameter-to-global-alias.zsh

```diff
@@ -28,8 +28,8 @@
 # vim: ft=zsh sw=2 ts=2 et
 # -------------------------------------------------------------------------------------------------
 
-alias -g x=y
-local s=x
+alias -g global_alias_which_hopefully_does_not_clash_with_an_external_command_name=y
+local s=global_alias_which_hopefully_does_not_clash_with_an_external_command_name
```

Comment:
Deployed a partial fix in 6d5372a937c63a565a1ff93aa6efa9a2b6179917.  If you'd like to improve things further, you're very welcome to do so, but do so in a new issue or pull request — do not overload this one.

Reply:
I don't like the solution of 6d5372a. This just skips the test now for no good reason.

Avoiding the name of the alias to be a command could at least be avoided by also choosing something other than `x` which is not as likely to clash.

Reply:
#761